app/models/my_module.rb

```diff
@@ -313,7 +313,7 @@ def repository_json_hot(repository, order)
     rows = repository.assigned_rows(self).includes(:created_by).order(created_at: order)
     rows.find_each do |row|
       row_json = []
-      row_json << (row.repository.is_a?(RepositorySnapshot) ? row.parent_id : row.id)
+      row_json << (row.repository.is_a?(RepositorySnapshot) ? row.parent.code : row.code)
```

Comment:
if parent is nil?

Reply:
I thought it cannot be nil if it's from a snapshot.

Reply:
snapshot of deleted inventory?

Reply:
fixed